katas/content/nonlocal_games/ghz_classical_strategy/index.md

```diff
@@ -0,0 +1,11 @@
+In this task you have to implement three functions, one for each player's classical strategy.
+Note that they are covered by one test, so you have to implement all of them to pass the test.
+For non-deterministic strategies, DrawRandomBoolean or [DrawRandomInt](https://learn.microsoft.com/qsharp/api/qsharp-lang/microsoft.quantum.random/drawrandomint) functions from Q# library can be used.
+
+**Inputs:**
+
+Alice, Bob, and Charlie's starting bits (X, Y, and Z).
```

Comment:
We called their input bits R, S, and T earlier in the kata, let's use the same notation throughout the game

Reply:
Done